store/src/java/com/zimbra/qa/unittest/SharedImapNotificationTests.java

```diff
@@ -359,6 +359,7 @@ protected String checkResult() throws Exception {
                     connection.fetch("1:*", "(ENVELOPE BODY)");
                     return "should not be able to connect; connection should be closed";
                 } catch (CommandFailedException e) {}
+                  catch (java.net.SocketException e) {}
```

Comment:
Wondering if it would be better to instead make CommandFailedException inherit from SocketException somewhere in the chain?

Reply:
@tmclane Personally, don't think so, the way my mind thinks, a command could fail for a number of reasons unrelated to sockets.
I'd suggest using the idiom:
`catch (CommandFailedException | java.net.SocketException e) {}`